src/modules/esl-utils/misc/array.ts

```diff
@@ -36,14 +38,8 @@ export function unwrap(value: any): any {
 }
 
 /** Makes array values unique */
-export const uniq = <T> (arr: T[]): T[] => {
-  if (arr.length < 2) return arr.slice(0);
-  const result: T[] = [];
-  const set = new Set<T>();
-  arr.forEach((item) => set.add(item));
-  set.forEach((item) => result.push(item));
-  return result;
-};
+export const uniq = <T> (arr: T[]): T[] =>
+  arr.length > 1 ? [...new Set<T>(arr)] : arr.slice(0);
```

Comment:
a'm not sure that it is rigth

Reply:
It looks really right!
The set will be created from an array and will not contain duplicate entries. Then spread on iteration over entries will create an array.

Reply:
It definitely looks right! 
The set will be created from an array and will not contain duplicate entries. Then spread on iteration over entries will create an array.

Reply:
Yes, I was confused by condition, but now i see that it's okay) 
Thanks!